### PR TITLE
Created workflow to run tests at simulated dates

### PR DESCRIPTION
**Pull request by maxirmx**

### CODE REVIEW

**Comment by maxirmx** [comment]
@ni4, I have created a workflow to run at simulated dates
It is unstable at '+1 year' and always fails at '+5 years'
What would we like to do with it ? 
I can look at the issues but not right now.

**Comment by ni4** [comment]
@maxirmx Thanks for this! I'll take care of issues, most likely those are related to exped keys, used in tests or something like that.

File: .github/workflows/time-machine.yml [hunk 1]
```diff
@@ -182,8 +182,7 @@ jobs:
         if:   matrix.date-offset != '0 years'
```
Comment (maxirmx) [inline .github/workflows/time-machine.yml:182]: @ni4  This condition is redundant after your changes

Comment (ni4) [reply .github/workflows/time-machine.yml]: @maxirmx Thanks for noticing, fixed.

**Review by ni4 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**